packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -7,16 +7,59 @@ import classNames from 'classnames';
 import { useFocusOnTriggeringElementOnClose } from '../../lib/hooks';
 import { Link, RouteComponentProps, useLocation } from '@reach/router';
 
+type ModalButtonProps = {
+  ctaText: string;
+  classname?: string;
+  revealModal: () => void;
+  modalRevealed?: boolean;
+  alertBarRevealed?: boolean;
+};
+
+export const ModalButton = ({
+  ctaText,
+  classname,
+  revealModal,
+  modalRevealed,
+  alertBarRevealed,
+}: ModalButtonProps) => {
+  const modalTriggerElement = useRef<HTMLButtonElement>(null);
+  // If the UnitRow children contains an AlertBar that is revealed,
+  // don't redirect focus back to the element that opened the modal
+  // because focus will be set in the AlertBar.
+  useFocusOnTriggeringElementOnClose(
+    modalRevealed,
+    modalTriggerElement,
+    alertBarRevealed
+  );
+
+  return (
+    <button
+      className={classNames(
+        'cta-neutral cta-base transition-standard ml-3',
```

Comment:
This looks good in Storybook and all the way up to tablet:
![image](https://user-images.githubusercontent.com/13018240/91334643-e2889780-e794-11ea-9f1b-c3d1582b61f0.png)


But at desktop, they do this: 
![image](https://user-images.githubusercontent.com/13018240/91334674-ed432c80-e794-11ea-90d0-f6ef38d2eec3.png)

I think we can fix this by wrapping each button in a `div` so they take up equal space in `unit-row-actions` instead of wrapping both buttons in a `div`.

Also, I think we need a new class that's not `cta-neutral`, that one's for grey buttons and the hover state is a little off. We need a new one for "caution" (that's what Figma calls it anyway), you could add them to `ctas.scss`:
![image](https://user-images.githubusercontent.com/13018240/91334860-31cec800-e795-11ea-9c4d-ee49e4d28904.png)

<img width="567" alt="image" src="https://user-images.githubusercontent.com/13018240/91334823-24b1d900-e795-11ea-9809-e76989fbd357.png">

Reply:
Adding to this, the mobile layout needs some margin work.

<img width="424" alt="Screen Shot 2020-08-26 at 2 21 31 PM" src="https://user-images.githubusercontent.com/6392049/91341396-75323200-e7a7-11ea-8724-b7e12417c3ea.png">